### PR TITLE
Simplify TV / Episode availability logic

### LINKED ISSUE #7059: Available episodes showing as not available
**Describe the bug**
We are displaying an episode is no longer available error where episodes are available but do not have an expiry date. 
This is because we display the expiry where availableUntil is not present in the endpoint. 
However, some episodes do not have an expiry date. 
Every available episode will have an availableFrom date, only some episodes versions will have no availableUntil Date. 

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'https://www.test.bbc.com/korean/bbc_korean_radio/w3ct0kfm'
2. Note the error 
3. Look at the availability flag in ARES endpoint: https://ares-api.api.bbci.co.uk/api/ws-media/radio/korean/bbc_korean_radio/w3ct0kfm
4. Note that the episode is available from the 26th of June 

**Expected behaviour**
Given episode w3ct0kfm is available from the 26th of June 2020
When I visit https://www.test.bbc.com/korean/bbc_korean_radio/w3ct0kfm'
then the episode is available to play

Given I visit brand page: https://www.test.bbc.com/korean/bbc_korean_radio/programmes/w13xttlm
then the lastest available episode is available to play 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Related to #7059 

**Overall change:**
Simplify the logic which determines whether an episode is available or not:
- if `versions` is empty, it has expired
- if `availableFrom` > now, it is not available yet
- otherwise, it is available

**Code changes:**
- Update logic to determine when an episode is available i.e. if versions array is empty

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/utilities/episodeAvailability/index.js [hunk 1]
```diff
@@ -28,9 +28,10 @@ const getEpisodeAvailability = pageData => {
     'availableFrom',
   ]);
 
+  const versions = path(['content', 'blocks', '0', 'versions'], pageData) || [];
```
Comment (ryanmccombe) [inline src/app/lib/utilities/episodeAvailability/index.js]:
```suggestion
  const episodeHasExpired = !path(['content', 'blocks', '0', 'versions', 0]);
```

File: src/app/lib/utilities/episodeAvailability/index.js [hunk 2]
```diff
@@ -28,9 +28,10 @@ const getEpisodeAvailability = pageData => {
...
   const availableFrom = getEpisodeAvailableFrom(pageData);
 
-  if (!availableFrom) {
+  if (versions.length === 0) {
```
Comment (ryanmccombe) [inline src/app/lib/utilities/episodeAvailability/index.js]:
```suggestion
  if (episodeHasExpired) {
```

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/lib/utilities/episodeAvailability/index.js (hunk 1 above)
Comment (karinathomasbbc) [reply src/app/lib/utilities/episodeAvailability/index.js]: This logic doesn't seem to work... maybe I am doing something wrong

**Comment by karinathomasbbc** [comment]
AReS may be changing how the episode availability is reported, so it makes sense to revisit this once that work has been completed

**Pull request closed without merging**